feature-examples/python/create_apikey.py

```diff
@@ -0,0 +1,129 @@
+"""Example of using the Vectara REST API to create an API Key."""
+
+import argparse
+import logging
+import requests
+import sys
```

Comment:
`requests` is third-party, so please move down to a new import block (standard imports only in first block, followed by third-party imports, followed by local imports, see https://peps.python.org/pep-0008/#imports).